src/main/java/uk/gov/hmcts/reform/pdf/service/exception/ExceptionHandling.java

```diff
@@ -24,14 +27,24 @@ public void handleException(Exception exception) {
     }
 
     @ExceptionHandler({
+        InvalidArgumentException.class,
+        MalformedTemplateException.class,
         MissingServletRequestParameterException.class,
         MissingServletRequestPartException.class,
-        InvalidArgumentException.class,
-        MalformedTemplateException.class })
+        ServletRequestBindingException.class })
     @ResponseStatus(code = HttpStatus.BAD_REQUEST)
     @ResponseBody
     public void handleMissingAndMalformedParametersValues(Exception exception) {
         log.error("Input parameters were missing/malformed:", exception);
     }
 
+    @ExceptionHandler(FeignException.class)
+    protected ResponseEntity handleFeignException(FeignException exc) {
+        return ResponseEntity.status(exc.status()).build();
+    }
+
+    @ExceptionHandler(AuthorisationException.class)
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
+    protected void handleAuthorisationException() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1186)